src/components/attendees/AttendeeTable.tsx

```diff
@@ -92,78 +88,48 @@ export default function AttendeeTable() {
 					}}
 				/>
 			</div>
+
 			{/* Table Component */}
-			<div className="h-full w-full overflow-auto  p-5">
-				<table className="w-full border-collapse border-spacing-0 p-5">
-					<tbody>
-						<tr>
-							<th>
-								<div id="nameContent">
-									Name
-									{!namesAscending ? (
-										<button>
-											<SortIcon id="ascendingSort" />
-										</button>
-									) : (
-										<button>
-											<SortIcon id="descendingSort" />
-										</button>
-									)}
-								</div>
-							</th>
-							<th> Email </th>
-							<th> Discord </th>
-							<th> Auth </th>
-							<th> School </th>
-							<th> Accepted </th>
-							<th> Confirmed </th>
-							<th> Checked In </th>
-							<th id="filterIcon" className="">
-								<div className=" flex w-full justify-end align-center">
-									<FilterIcon />
-								</div>
-							</th>
-						</tr>
+			<Table
+				headerContent={
+					<AttendeeTableHeader
+						namesAscending={namesAscending}
+						setNamesAscending={(namesAscending) => {
+							setNamesAscending(namesAscending);
+						}}
+					/>
+				}
+				bodyContent={
+					<>
 						{attendeeData.map((attendee) => {
 							return (
 								<tr key={attendee.id}>
 									<AttendeeRow
 										key={attendee.id}
 										attendee={attendee}
+										popUp={popUp}
+										setOpen={(isOpen) => {
+											if (!popUp.seenAttendeePopUp) {
+												setPopUp({
```

Comment:
If this didn't get fixed in the previous PR, this can still remove the conditional and just be:
```
setPopUp((popUp) => ({
    ...popUp,
    seenAttendeePopUp: true,
}));
```